packages/core/src/dispatcher.ts

```diff
@@ -126,13 +126,10 @@ export class Dispatcher {
     // TODO Add validation the message adheres to the proper format.
 
     const { doc: docId, tip } = message
-    const document = await this.repository.get(docId)
-    if (document) {
-      // TODO: add cache of cids here so that we don't emit event
-      // multiple times if we get the message more than once.
-      this.repository.stateManager.update(document, tip)
-      // TODO: Handle 'anchorService' if present in message
-    }
+    // TODO: add cache of cids here so that we don't emit event
+    // multiple times if we get the message more than once.
```

Comment:
```suggestion
    // TODO: add cache of cids here so that we don't emit event
    // multiple times if we get the message from more than one peer.
```